triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/ServiceTest.scala

```diff
@@ -189,6 +189,7 @@ class ServiceTest extends AsyncFlatSpec with Eventually with Matchers {
       resp <- startTrigger(uri, s"$testPkgId:TestTrigger:trigger", "Alice")
       triggerId <- parseTriggerId(resp)
 
+      _ <- Future { Thread.sleep(1000); true }
```

Comment:
That is a great way to write a flaky test. Please use `eventually` instead.

Reply:
I'll look into `eventually`.

Reply:
This is fixed now.